index.html

```diff
@@ -25,6 +25,10 @@
     <a style="margin-right: 1em;" href="https://github.com/Bitcoin-ABC/bitcoin-abc">Source Code</a>
     <a style="margin-left: 1em;" href="https://download.bitcoinabc.org/">Binaries</a>
   </div>
+  
+  <!–– Twitter feed embed code ––>
+  <a class="twitter-timeline" data-height="450" href="https://twitter.com/Bitcoin_ABC?ref_src=twsrc%5Etfw">Tweets by Bitcoin_ABC</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
```

Comment:
Put the <script> tag on its own line.

Reply:
OK, I made the change.
tested again locally, still looks the same.